lib/tasks/argo.rake

```diff
@@ -17,14 +17,17 @@ end
 task :default => [:ci]
 
 task :ci do
-  ENV['RAILS_ENV'] = 'test'
-  Rake::Task['argo:install'].invoke
-  jetty_params = jettywrapper_load_config()
-  error = Jettywrapper.wrap(jetty_params) do
-    Rake::Task['argo:repo:load'].invoke  # load 'em all!
-    Rake::Task['spec'].invoke
+  if Rails.env.test?
+    Rake::Task['argo:install'].invoke
+    jetty_params = jettywrapper_load_config()
+    error = Jettywrapper.wrap(jetty_params) do
+      Rake::Task['argo:repo:load'].invoke  # load 'em all!
+      Rake::Task['spec'].invoke
+    end
+    raise "test failures: #{error}" if error
+  else
```

Comment:
I hear you on that. The intent of this PR was to implement the consensus that I thought was gained from #172. Happy to close this down, if we don't have that concensus.